src/ifElse.test.js

```diff
@@ -1,11 +1,57 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
```

Comment:
The test case is incomplete. There is no description of the test case. It's not clear what should be tested in this case.